### ISSUE TITLE
crashing when attempting to view the Animus tab

### ISSUE THREAD

**Issue by Hezkezl** [VarsityPhysicist/FFXIVRelicTracker#5]
[Characters.txt](https://github.com/VarsityPhysicist/FFXIVRelicTracker/files/5345330/Characters.txt)

Changed my server name to Nunya and removed my character's last name, just in case :D 

Hopefully this will help you figure out why it's crashing sometimes! I can provide a screen recording of the crash sequence (what I do right before hand to make it crash) if required.

**Comment by Hezkezl** [comment]
Did some more digging. It looks like the crashing happens when you have an Animus book that only has the dungeons left, and then you leave the Animus tab and try to go back to it.

If all the Beastmen, FATEs, and Leves are marked as completed, it will remove all of the 'Available Maps' info and not show any maps on the right side (because the dungeons aren't shown on maps.) Then unselecting the Animus tab (aka: clicking any other tab) and then trying to go back to it will crash the program. It doesn't matter what book is selected, or what the active job is, or what progress is done on anything else in the program. I tested several combinations and it only seemed to happen when I had a book that didn't have anything besides dungeons left on it, and then I tried to go back to the tab :D
